phlib/include/treenew.h

```diff
@@ -10,6 +10,8 @@ extern "C" {
 #define PH_TREENEW_SEARCH_TIMEOUT 1000
 #define PH_TREENEW_SEARCH_MAXIMUM_LENGTH 1023
 
+#define PH_TREENEW_FIXED_WIDTH_FONT 0x80000000
```

Comment:
I suggest moving this next to the other `PH_ALIGN_xxx` defines in `guisup.h` name it with the same prefix, suggesting `PH_ALIGN_MONOSPACE_FONT`.